deploy.sh

```diff
@@ -42,6 +42,7 @@ sam deploy --stack-name "$stack_name" \
    --capabilities CAPABILITY_IAM \
    --parameter-overrides \
    CodeSigningEnabled=false \
+   DevEnvironment=not-cri-dev \
```

Comment:
Worth updating the readme to call out that deploy.sh is for local deploys to dev envs in case someone tries to reuse this script

Reply:
Added to readme